### PR TITLE
pkp/pkp-lib#3578: harmonizing GridCellProvider::getCellActions() signatures

### PR DESCRIPTION
**Pull request by crism**
Fixes part of #3578; may break some plugins.

### CODE REVIEW

File: controllers/grid/queries/QueryNotesGridCellProvider.inc.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ function getCellActions($request, $row, $column) {
 				}
 				return $actions;
 		}
-		return parent::getCellActions($request, $row, $column);
+		return parent::getCellActions($request, $row,, $column, $position);
```
Comment (asmecher) [inline controllers/grid/queries/QueryNotesGridCellProvider.inc.php]: This looks like a typo.

Comment (asmecher) [reply controllers/grid/queries/QueryNotesGridCellProvider.inc.php]: @crism, it looks like the tests have passed on the three PRs here, but I can't merge until this comment gets resolved ^

Comment (crism) [reply controllers/grid/queries/QueryNotesGridCellProvider.inc.php]: Sorry, I looked for that before sending the PR, and still somehow missed it.

Comment (crism) [reply controllers/grid/queries/QueryNotesGridCellProvider.inc.php]: Fixed.

**Comment by asmecher** [comment]
Merged, thanks!

**Pull request merged**